scripts/globals/spells/damage_spell.lua

```diff
@@ -616,6 +661,14 @@ xi.spells.damage.calculateMagicBonusDiff = function(caster, target, spellId, ski
         mab = mab + caster:getMod(xi.mod.NIN_NUKE_BONUS_INNIN)
     end
 
+    -- WHM Catagory 1 Merits (Banish)
+    if
+        (spellId >= xi.magic.spell.BANISH and spellId <= xi.magic.spell.BANISH_V) or
+        (spellId >= xi.magic.spell.BANISHGA and spellId <= xi.magic.spell.BANISHGA_V)
+    then
+        mab = mab + (caster:getMerit(xi.merit.BANISH_EFFECT) * 2) -- +2 Magic Attack Bonus for each merit level.
+    end
+
```

Comment:
Banish merits do not give MAB. It just add 2 to base dmg per merit

Reply:
also keep in mind that merits have a value, which defines what you get per merit point:

![image](https://github.com/LandSandBoat/server/assets/131182600/5585aef7-a2b7-484c-bb18-90a25055d81b)

5 points in banish effect would return `caster:getMerit(xi.merit.BANISH_EFFECT) == 10`